_scripts/webpack.renderer.config.js

```diff
@@ -134,8 +136,6 @@ const config = {
   ],
   resolve: {
     alias: {
```

Comment:
Looks like webpack is picking the wrong export for the electron renderer build, it's picking the correct one for the web build though, guessing the combined web+node environment confuses it.
![very-wrong-vue](https://github.com/FreeTubeApp/FreeTube/assets/48293849/aecaaf8f-52db-4165-956d-80cff4528100)
![wrong-vue](https://github.com/FreeTubeApp/FreeTube/assets/48293849/84123d41-19dc-42e1-a5ab-e1daa95c63bd)




```suggestion
    alias: {
      vue$: 'vue/dist/vue.runtime.esm-bundler.js',

```